formwidgets/menuitems/assets/js/menu-items-editor.js

```diff
@@ -183,11 +183,23 @@
             }
         }
 
+        var defaultLocale = $('[data-control="multilingual"]').data('default-locale')
```

Comment:
What happens with this line of code when RainLab.Translate is not present on the system?

Reply:
defaultLocale becomes null

Reply:
the true answer is "undefined". Which is ok, since all other jquery selector will not find locale related elements.

Reply:
I tested with the Translate plugin disabled and previous behavior is maintained.

Reply:
Can you test with the browser console open and make sure that there are no console errors?

Reply:
I did already, but I can double check.

Reply:
I confirm @LukeTowers , both with Translate Plugin enabled and disabled, NO javascript errors when interacting with the Menu / MenuItem.